tests/beaker_tests/netdev_stdlib/port_channel_provider_defaults.rb

```diff
@@ -161,7 +161,12 @@ def find_ethernet_interface(tests, index)
 
```

Comment:
Please move out of the `netdev_stdlib` test folder into `tests/beaker_tests/port_channel` and renaming to `test_port_channel.rb` to reflect that this is a new style test.

Reply:
Moved

Reply:
sorry but looks like this has been moved to `tests/port_channel` rather than `tests/beaker_tests/port_channel`